### PR TITLE
introduce runc versioning via nodebootstrapingconfig

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -161,18 +151,28 @@ installMoby() {
         fi
         apt_get_install 20 30 120 moby-engine=${MOBY_VERSION}* moby-cli=${MOBY_CLI}* --allow-downgrades || exit $ERR_MOBY_INSTALL_TIMEOUT
     fi
-    ensureRunc
+    ensureRunc ${RUNC_VERSION:-""}
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:154]: can we have the caller pass in the runc version?

Comment (eriksywu) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:154]: caller of installContainerd/installDocker?

File: pkg/agent/datamodel/types.go [hunk 2]
```diff
@@ -1424,6 +1424,7 @@ type NodeBootstrappingConfiguration struct {
 	Enable1804Chrony              bool
 	TeleportdPluginURL            string
 	ContainerdVersion             string
+	RuncVersion					  string
```
Comment (xuto2) [inline pkg/agent/datamodel/types.go:1427]: nit: extra space

**Comment by xuto2** [comment]
lgtm

**Review by xuto2 — APPROVED** [review]

**Pull request merged**